salt/modules/localemod.py

```diff
@@ -114,6 +116,8 @@ def get_locale():
     '''
     Get the current system locale
 
+    .. versionchanged:: Boron
```

Comment:
I'm not sure if this is useful without any indication of *what* changed. I would just remove it. The only time we need this is when there's a backwards incompatible change anyway.